<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendfile</class>
 <widget class="QDialog" name="sendfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送文件</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sOpenBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sSendBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sCloseBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="sStatuLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要发送的文件</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>64B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>512B</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sStatuLbl_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择数据包的大小</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
